### PR TITLE
release(v1.3.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.3.0-alpha.0 release.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: hack/release.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ github_repo = "siderolabs/talos"
 match_deps = "^github.com/((talos-systems|siderolabs)/[a-zA-Z0-9-]+)$"
 
 # previous release
-previous = "v1.2.2"
+previous = "v1.2.0"
```
Comment (DmitriyMV) [inline hack/release.toml:9]: Just curious - why is it v1.2.0 and not v1.2.2?

**Review by DmitriyMV — APPROVED** [review]

File: hack/release.toml (hunk 1 above)
Comment (smira) [reply hack/release.toml:9]: Right now the latest is v1.2.3, as we release 1.2.4, it will be 1.2.4, and so on.

**Pull request merged**